lib/android/audio.nit

```diff
@@ -377,6 +377,11 @@ class MediaPlayer
 
 	# Load a sound for a given resource id
 	fun load_sound(id: Int, context: NativeActivity): Music do
+		if playing then
+			stop
+			reset
```

Comment:
It resets the state of the object as if it was just created, so it's to be sure that resources are cleared before removing completely the native media player associated with it.